docs/FAQ.md

```diff
@@ -11,4 +11,5 @@
 
 ### AttributeError: module 'tornado.web' has no attribute 'asynchronous'
 
-TabPy uses Tornado 5.1.1. To it to your Python environment run `pip install tornado==5.1.1` and then try to start TabPy again.
+TabPy uses Tornado 5.1.1. To it to your Python environment run
```

Comment:
If we are no longer bound to 5.1.1 should we remove this from the documentation?

Reply:
Yes, we should. But I'll keep it for now for 2 reasons:
1. People may still have the issue using an older version of TabPy, and
2. The file itself will be empty before we add any other Q/A there :)